docs/testing.mdx

```diff
@@ -16,10 +16,10 @@ By default snapshots with emotion show generated class names. Adding [jest-emoti
 npm install --save-dev jest-emotion
 ```
 
-Add the snapshot serializer in your [`setupTestFrameworkScriptFile`](http://facebook.github.io/jest/docs/en/configuration.html#setuptestframeworkscriptfile-string) _or_ at the top of your test file.
+Add the snapshot serializer in your [`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array) _or_ at the top of your test file.
 
 ```javascript
-import * as emotion from 'emotion'
+import * as emotion from '@emotion/core'
```

Comment:
Typescript complains for me using this line as the emotion object with `createSerializer`. (although it also complains with the old syntax). Is `createSerializer` typed properly?

Reply:
It should be typed correctly - could you provide a repro case?

Reply:
Related: 
![Screen Shot 2020-06-11 at 8 40 12 pm](https://user-images.githubusercontent.com/13936670/84376322-500c3980-ac24-11ea-977b-e6a0f7087eca.png)

Reply:
it's reproducible in this repo: https://github.com/DarkPurple141/jest-emotion-typescript-minimal/blob/master/setup.ts

Reply:
Please always provide the issue **reproduced**. I'm really trying my best to provide help quickly and the linked file doesn't look like the one on your screenshot. I had to see a working file without any error and then jump here back to decipher what you were exactly trying to do. Context switches hurt and slow down.

This proposed change is incorrect - you shouldn't pass emotion to the `createSerializer`. It only accepts those 2 options:
https://github.com/emotion-js/emotion/blob/42714ccc080e74b5a2e44be97b0238c14c78a11a/packages/jest-emotion/README.md#options

I don't think you really should create your own serializer, just use the builtin one.

Reply:
That pattern is from the [offical docs](https://emotion.sh/docs/testing#installation) which is what this PR is ultimately about tackling.

Reply:
Right, I will handle correcting it soon.